src/components/Drawer.svelte

```diff
@@ -1,17 +1,10 @@
 <div ref:wrapper class='coz-drawer-wrapper' on:click='fire("close")'>
   <aside ref:aside on:click='event.stopPropagation()'>
     <nav class='coz-drawer--apps'>
-      <h1>{{t('drawer apps')}}</h1>
-      {{#if splitContentsArrays.length < 2}}
-        <NavigationGroup group='{{content}}' itemsLimit={{3}} />
-      {{else}}
-        <div class='coz-drawer--apps-paginator'>
-          {{#each splitContentsArrays as subgroup}}
-            <NavigationGroup group='{{subgroup}}' itemsLimit={{3}} />
-          {{/each}}
-        </div>
-      {{/if}}
-
+      {{#each categories as category}}
+        <h2 class='coz-nav-category'>{{t(`Categories.${category.title}`)}}</h2>
```

Comment:
I used `h2` because `h1` is already use for application name in the bar and to be more consistent with the [`NavigationSection`](https://github.com/cozy/cozy-bar/pull/52/files#diff-8f646533758ab8608cfd8140b4c1297cR18)